dimagi/utils/django/email.py

```diff
@@ -30,6 +33,14 @@ def send_HTML_email(subject, recipient, html_content, text_content=None,
                          "\n {}".format(html_content)
             )
 
+    if ga_track:
+        url = "https://www.google-analytics.com/collect?v=1&tid={ga_tid}&cid={ga_cid}&t=event&ec=email".format(
+            ga_tid=settings.ANALYTICS_IDS['GOOGLE_ANALYTICS_ID'],
+            ga_cid=uuid.uuid4().hex)
+        new_content = '<img src="{url}&ea=open"/>\n</body>'.format(url=url)
+        html_content, count = re.subn(r'(.*)</body>', r'\1'+new_content, html_content)
+        if count == 0:
+            raise Exception('Cannot add tracking to HTML Email with no closing body tag')
```

Comment:
@czue I assume we should never hit this because our emails use valid html but it was easy enough to add that the benefits you mentioned seem worth it